frontend/src/scss/global.scss

```diff
@@ -5,45 +5,72 @@ a {
     text-decoration: none;
 }
 
-#title {
-    font-family: $font-header;
-    font-size: 42px;
-    position: relative;
-    padding-bottom: 3rem;
+// Container classes
+
+.blue-container,
+.white-container {
+    background: $blue;
+    padding: 2rem 6rem;
```

Comment:
I don't like the idea of defining these two classes together since they have only common padding. And you have to override color of `white-container`.